### DISCUSSION TITLE
diff でコードスニペットを書いたところをコピーしたときに「＋」をコピーしないでほしい

### DISCUSSION THREAD

**Discussion by duri0214** [increments/qiita-discussions#325] [要望・アイデア・不具合]
![image](https://user-images.githubusercontent.com/19668980/195845822-0066a3c7-a2cf-458f-b373-33266484388e.png)
記事を公開したあと、その記事をもとに人にレクチャーしていると「＋」を巻き込んでコピペして「動かない」というひとが本当にいる。。（つらい）
可能であれば、diff で「＋」や「ー」を書いたところはコピーされないようにしてほしい

**Comment by tanaka-asami415** [comment]
@duri0214 
ご意見ありがとうございます🙏検討します！

**Comment by kyntk** [comment]
@duri0214 
取り急ぎ、Chrome、Firefox、Edgeで選択されないように変更してみました！

**Reply by duri0214** [reply]
早い対応助かります

**Reply by duri0214** [reply]
@kyntk 赤い「＋」と白い「＋」があることに気が付きました（白い方は「＋」を巻き込んでしまいます）。とくに [markdownの解説のページ](https://help.qiita.com/ja/articles/qiita-markdown) を見てもdiffについては書かれていないようです。この白い「＋」については、赤い「＋」に書く方法がわかればこちらで運用回避できると思います
![image](https://user-images.githubusercontent.com/19668980/197309914-81e87244-8c67-44d2-a0bd-599f6362750d.png)

**Reply by duri0214** [reply]
思い出した。diffのなかの `+` や `=` が赤くなるとコピーできないこともわかりました
![image](https://user-images.githubusercontent.com/19668980/197312021-5a0646f9-4f2f-4dda-812a-27506837a683.png)

**Comment by kyntk** [comment]
@duri0214
ご報告ありがとうございます。

不具合に関して申し訳ございません。
調査したのですが、今回の変更とは別でハイライトの処理を修正する必要があり、こちらはすぐの修正が難しいため、一旦変更を切り戻させていただきます :sweat_drops: :bow:

また、工数との兼ね合いから、正式な対応は先になってしまうと思います。:bow:

**Reply by duri0214** [reply]
@kyntk （チケットとして）可視化されていれば大丈夫です。よろしくお願いいたします
